src/app/component/ProductCard/ProductCard.component.js

```diff
@@ -55,6 +58,22 @@ export default class ProductCard extends PureComponent {
 
     imageRef = createRef();
 
+    handleClick = () => {
```

Comment:
Shange it to simple method, as it rise problem in method rewriting from child theme